script.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const player1El = document.querySelector(".player--1");
+const player2El = document.querySelector(".player--2");
+const score1El = document.querySelector("#score--1");
+const score2El = document.getElementById("score--2");
+const current1El = document.getElementById('current--1');
+const current2El = document.getElementById('current--2');
+
+const diceEl = document.querySelector('.dice');
+const btnNew = document.querySelector('.btn--new');
+const btnRoll = document.querySelector('.btn--roll');
+const btnHold = document.querySelector('.btn--hold');
```

Comment:
Can we use `querySelectorAll` for this ?

Reply:
Yes ! But now just to keep it simple I have selected separately.